fail2ban/server/filterpyinotify.py

```diff
@@ -113,14 +151,81 @@ def _process_file(self, path):
 			self.failManager.cleanup(MyTime.time())
 		self.__modified = False
 
+	def _addPending(self, path, reason, isDir=False):
+		if path not in self.__pending:
+			self.__pending[path] = [self.sleeptime / 10, isDir];
+			self.__pendingNextTime = 0
+			if isinstance(reason, pyinotify.Event):
+				reason = [reason.maskname, reason.pathname]
+			logSys.log(logging.MSG, "Log absence detected (possibly rotation) for %s, reason: %s of %s",
+				path, *reason)
```

Comment:
reason could be that event passed in... Is it a named tuple of guaranteed length 2?